public/components/common/helpers/ppl_docs/overview.tsx

```diff
@@ -5,33 +5,22 @@
-queries and explore user data. SQL allows users to extract and analyze
-data in OpenSearch in a declarative manner. OpenSearch now makes its
-search and query engine robust by introducing Piped Processing Language
-(PPL). It enables users to extract insights from OpenSearch with a
-sequence of commands delimited by pipes () syntax. It enables
-developers, DevOps engineers, support engineers, site reliability
-engineers (SREs), and IT managers to effectively discover and explore
-log, monitoring and observability data stored in OpenSearch.
-
-We expand the capabilities of our Workbench, a comprehensive and
-integrated visual query tool currently supporting only SQL, to run
-on-demand PPL commands, and view and save results as text and JSON. We
-also add a new interactive standalone command line tool, the PPL CLI, to
-run on-demand PPL commands, and view and save results as text and JSON.
-
-The query start with search command and then flowing a set of command
-delimited by pipe ( for example, the following query retrieve firstname
-and lastname from accounts if age large than 18.
+Piped Processing Language (PPL) is a query language for OpenSearch processes data in a sequential, step-by-step manner. PPL uses a set of commands, connected by pipes (|), to process data and return results. (Note that the requests and results are read only.) 
+
+You can query data in OpenSearch using <a href="{{https://opensearch.org/docs/latest/search-plugins/sql/ppl/index/}}">PPL</a>, <a href="{{https://opensearch.org/docs/latest/query-dsl/index/}}">Query DSL</a>, or <a href="{{https://opensearch.org/docs/latest/search-plugins/sql/sql/index/}}">SQL</a>.
+
+PPL is designed to be easy to use. You don't need to learn a complex syntax or understand how OpenSearch works behind the scene. Just write the commands you need, and PPL will take care of the rest. 
+
+PPL is the primary language used with observability in OpenSearch, and it supports multi-data queries. Developers, DevOps engineers, support engineers, site reliability engineers (SREs), and IT managers find it useful for exploring and discovering log, monitoring, and observability data. For example, you can use PPL to:
+
+- Find all log messages that contain a specific error code. 
+- Identify trends in your data over time. 
+- Group similar data points.
+- Caluculate statistics on your data.
+
+PPL is availble in OpenSearch Dashboards and in a standalone command-line tool. Within OpenSearch Dashboards, you can use <a href="{{https://opensearch.org/docs/latest/dashboards/query-workbench/}}">Query Workbench</a> to run on-demand PPL commands and view and save the results as text and JSON. The <a href="{{https://opensearch.org/docs/latest/search-plugins/sql/cli/}}">PPL command line interface (CLI)</a> is a standalone Python application that you can launch with the \`opensearchsql\` command. The PPL CLI is useful for running on-demand PPL commands and viewing and saving the results as text and JSON.
```

Comment:
Please confirm the inline hyperlink formatting is correct. If not, provide an example in the proper format.

Reply:
This looks like markdown, so both `[annotation](link)` and `<a href="link">annotation</a>` should work, though `[]()` is more compatible with safe markdown environments that block inline html.